commands/operator-sdk/cmd/up/local.go

```diff
@@ -174,6 +175,20 @@ func upLocalHelm() error {
 	return helm.Run(helmOperatorFlags)
 }
 
+func buildLocal(outputBinName string) error {
+	args := []string{"build", "-o", outputBinName}
+	if ldFlags != "" {
+		args = append(args, []string{"-ldflags", ldFlags}...)
```

Comment:
I believe it's not needed to create the []string. you should be able to `append(args, "-ldflags", ldFlags)`